sass/base/_structure.scss

```diff
@@ -22,7 +22,7 @@ body {
     }
 }
 
-div:focus {
+*:focus {
```

Comment:
Are we manually providing the visual focus for all cases that need focus, not just those handled by the new accessibility functionality?

Reply:
We do not need to apply focus if a user is using a mouse, and if a user is using a tab, then yes, all the elements in the app should ideally get the `keyboard-focus` class, as that is tied to the `tab` keyup.

Reply:
I have some best-practice-related concerns about brute-force removal of visual focus indicators, even for mouse usage. That said, it's not easily addressed in the current state and given it's the current practice for the webapp and we are at least adding visual cues for keyboard navigation with the current accessibility effort, I'm okay running with this.